src/_includes/navigation-main.html

```diff
@@ -14,8 +14,8 @@
         {%- if page_url_path contains '/*/guides/' or page_url_path contains '/*/dart-2/' or page_url_path contains '/*/codelabs/' or page_url_path contains '/*/samples/' or page_url_path contains '/*/tools/' or page_url_path contains '/*/tutorials/' or page_url_path contains '/*/server/' or page_url_path contains '/*/web/' %} active {%- endif -%}"><span>Docs</span></a>
     </li>
     <li>
-      <a href="/platforms" class="nav-link
-        {%- if page_url_path contains '/*/platforms/' %} active {%- endif -%}">Platforms</a>
+      <a href="/overview" class="nav-link
+        {%- if page_url_path contains '/*/overview/' %} active {%- endif -%}">Overview</a>
```

Comment:
Maybe we should swap Overview and Docs? It makes sense to me to have Overview first.